.github/workflows/manual-deploy-testnet-faucet.yml

```diff
@@ -32,7 +32,19 @@ on:
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
+    environment:
+      name: ${{ github.event.inputs.testnet_type }}
     steps:
+      - name: 'Print GitHub variables'
+        # This is a useful record of what the environment variables were at the time the job ran, for debugging and reference
+        run: |
+          echo "ACCOUNT_ADDR_WORKER=${{ vars.ACCOUNT_ADDR_WORKER }}"
+          echo "ACCOUNT_ADDR_NODE_0=${{ vars.ACCOUNT_ADDR_NODE_0 }}"
+          echo "ACCOUNT_ADDR_NODE_1=${{ vars.ACCOUNT_ADDR_NODE_1 }}"
+          echo "DOCKER_BUILD_TAG_FAUCET=${{ vars.DOCKER_BUILD_TAG_FAUCET }}"
+          echo "FAUCET_PAY_AMOUNT=${{ vars.FAUCET_PAY_AMOUNT }}"
+          echo "L2_RPC_URL_VALIDATOR=${{ vars.L2_RPC_URL_VALIDATOR }}"
```

Comment:
Nice, yeah I did briefly look into this but didn't think it was possible. Seems like I didn't look hard enough though haha, will figure out how to get it working because agree it's a maintenance nightmare.

Reply:
Yeah the change I've added does the trick I think, pretty prints them in alphabetical order:
<img width="900" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/98158711/c54bd11d-961c-4396-a270-225740565ef8">